src/main/kotlin/com/cognifide/gradle/aem/instance/action/AwaitAction.kt

```diff
@@ -84,6 +86,42 @@ open class AwaitAction(project: Project, val instances: List<Instance>) : Abstra
         progressLogger.completed()
     }
 
+    private fun prepareSynchronizers(): List<InstanceSync> {
+        return instances.map { instance ->
+            val init = instance is LocalInstance && !LocalHandle(project, instance).initialized
+
+            InstanceSync(project, instance).apply {
+                val sync = this
+
+                requestConfigurer = { request ->
+                    request.config = RequestConfig.custom()
+                            .setConnectTimeout(timeout)
+                            .setSocketTimeout(timeout)
+                            .build()
+                }
+
+                if (init) {
+                    sync.basicUser = Instance.USER_DEFAULT
+                    sync.basicPassword = Instance.PASSWORD_DEFAULT
+                }
+
+                responseHandler = { response ->
+                    if (init) {
+                        if (response.statusLine.statusCode == HttpStatus.SC_UNAUTHORIZED) {
```

Comment:
fallback mechanism, if instance is being initialized (running up first time), then on authorized we are switching creds to target ones